internal/pkg/admission/reconciler.go

```diff
@@ -280,14 +316,6 @@ func (r *Reconciler) enablePolicyWebhook(
 	policyServer *policiesv1alpha2.PolicyServer,
 	policyServerSecret *corev1.Secret,
 	clusterAdmissionPolicies *policiesv1alpha2.ClusterAdmissionPolicyList) error {
-	policyServerReady, err := r.isPolicyServerReady(ctx, policyServer)
```

Comment:
But note that this code is inside `reconciler.enablePolicyWebhook()`. This is the code that takes care of registering the webhooks that tie the policies with the PolicyServer that has them deployed (This happens after the PolicyServer deployment has been updated so it contains a policy-server for the policies specified, and after the PolicyServer deployment is ready to serve). We need to keep checking `isPolicyServerReady()` before registering the policies (which is what ultimately makes them active). If we don't check `isPolicyServerReady()`, we could be registering faulty policies, or policies that don't have a PolicyServer. And depending on the resources the policies are tracking, this could mean no new resources could be instantiated, not even for fixing the policies or the PolicyServer. Effectively giving us a denial of service of the cluster.